jobbergate-api/jobbergate_api/tests/apps/job_scripts/conftest.py

```diff
@@ -59,19 +57,3 @@ def template_files(template_files_application_content):
     Provide a fixture that creates test template files.
     """
     return {"application.sh": template_files_application_content}
-
-
-@fixture
-def s3_object():
-    """
-    Provide a fixture that creates a test s3 object.
-    """
-    return {"Body": open("jobbergate_api/tests/apps/job_scripts/test_files/jobbergate.tar.gz", "rb")}
```

Comment:
These fixtures were deleted because they were not used anywhere. In this case, we should consider deleting the file mentioned here as well.

Reply:
Actually, the fixtures were moved to another file.